src/content/contributing/community-membership/_index.en.md

```diff
@@ -186,8 +194,10 @@ The following apply to people who would be an owner:
 [contributor guide]: https://submariner-io.github.io/en/contributing/
```

Comment:
Not related to this change, but with Spanish support removed we should not use /en anymore and this link is broken. Should be https://submariner-io.github.io/contributing/

Reply:
Good catch, fixed that example and another one here: https://github.com/submariner-io/submariner-website/pull/87